### PR TITLE
Replaces gRPC Printer with REST API

### PR DESCRIPTION
**Pull request by allen-tran**
We now have an endpoint with the logging service that accepts uploaded pdfs from Core-v4.

### CODE REVIEW

**Comment by allen-tran** [comment]
Evan will test this on Friday in the room.

File: src/APIFunctions/2DPrinting.js [hunk 1]
```diff
@@ -27,11 +28,13 @@ export const range = (start, end) => {
...
 export async function healthCheck() {
   let status = new ApiResponse();
-  await axios.post(RPC_API_URL + '/Printer/healthCheck')
+  await axios.post(LOGGING_API_URL + '/Printer/healthCheck')
     .then(res => {
+      console.error('sup cutty', res.data);
       status.reponseData = res.data;
     })
     .catch(err => {
+      console.error('i am so mad now');
```
Comment (evanugarte) [inline src/APIFunctions/2DPrinting.js:37]:
```suggestion
      status.reponseData = res.data;
    })
    .catch(err => {
```

File: src/APIFunctions/2DPrinting.js [hunk 2]
```diff
@@ -80,8 +83,9 @@ export function parseRange(pages, maxPages) {
  * the page successfully printed
  */
 export async function printPage(data) {
+  console.log({data})
```
Comment (evanugarte) [inline src/APIFunctions/2DPrinting.js:86]:
```suggestion
```

File: package.json [hunk 3]
```diff
@@ -56,6 +56,7 @@
     "filepond-plugin-image-preview": "^4.6.5",
     "google-spreadsheet": "^3.0.13",
     "googleapis": "^67.1.0",
+    "inquirer": "^8.1.5",
```
Comment (evanugarte) [inline package.json:59]: missed this, lets remove together + rearrange some other files

**Comment by allen-tran** [comment]
Outdated duplicate of #907. No longer needed. 
The birth of [Quasar](https://github.com/SCE-Development/Quasar) enables VM SQS reading while #907 & #908 is set to handle S3 uploading and SQS writing.

**Pull request closed without merging**